doc/filtered_replication.md

```diff
@@ -0,0 +1,59 @@
+Filtered Replication/Changes
+============================
+
+The pouchdb documentation is really good to understand it:
+
+ * [API for filtered replication][1]
+ * [API for filtered changes][2]
+ * [Blog post for filtered replication][3]
+
+The goal is to filter couch docs in remote not in local device.
+
+
+Configuration
+-------------
+
+The configuration allow you what you want synchronize:
+
+```javascript
+config = {
+    file: true,
+    folder: true,
```

Comment:
I think we should do the same for file that for calendar, ie get the associated docTypes. With, `calendar: true`, the docTypes `event` and `tag` are kept. With `file: true`, I think we should keep `file` and `folder` docTypes. What do you think?